docs/sources/helm-charts/mimir-distributed/configure/configure-autoscaling.md

```diff
@@ -0,0 +1,194 @@
+---
+description: Learn how to configure Grafana Mimir autoscaling when using Helm.
+menuTitle: Configure autoscaling
+title: Configure Grafana Mimir autoscaling with Helm
+weight: 30
+---
+
+# Configure Grafana Mimir autoscaling with Helm
+
+{{< admonition type="warning" >}}
+Autoscaling support in the Helm chart is currently experimental. Use with caution in production environments and thoroughly test in a non-production environment first.
+{{< /admonition >}}
+
+This topic explains how to configure autoscaling for Mimir components using KEDA and Kubernetes Horizontal Pod Autoscaler (HPA).
+
+## Before you begin
+
+- Ensure you have a running Mimir cluster deployed with Helm
+- Verify you have the required permissions to modify Helm deployments
+- Familiarize yourself with Kubernetes HPA concepts
+
+## Prerequisites
+
+To use autoscaling, you need:
+
+1. KEDA installed in your Kubernetes cluster
+2. Prometheus metrics available for scaling decisions
+
+{{< admonition type="warning" >}}
+Do not use the same Mimir or Grafana Enterprise Metrics cluster for storing and querying autoscaling metrics. Using the same cluster can create a dangerous feedback loop:
+
+1. If the Mimir/GEM cluster becomes unavailable, autoscaling will stop working because it cannot query the metrics
+2. This can prevent the cluster from automatically scaling up during high load or recovery
+3. The inability to scale can further exacerbate the cluster's unavailability
+4. This may prevent the cluster from recovering
+
+Instead, use a separate Prometheus instance or a different metrics backend for autoscaling metrics.
+{{< /admonition >}}
+
+## Supported components
+
+Mimir Helm chart supports autoscaling for the following components:
+
+- [Querier](https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/querier/)
+- [Query-frontend](https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/query-frontend/)
+- [Distributor](https://grafana.com/docs/mimir/<MIMIR_VERSION>/references/architecture/components/distributor/)
+
+## About KEDA
+
+KEDA is a Kubernetes operator that simplifies the setup of HPA with custom metrics from Prometheus. It consists of:
```

Comment:
Is KEDA an abbreviation for something? If so, we should define it the first time we use it, i.e. "Whatever It Stands For (KEDA)".

Reply:
Same with HPA 😄

Reply:
we do it for HPA earlier in the doc and I modified your suggestion to introduce KEDA too